gm4_mending_tanks/data/mending_tanks/functions/item_fill.mcfunction

```diff
@@ -0,0 +1,6 @@
+#@s = liquid tank with item in first slot
+#run from liquid_tanks:item_process
+
+
+#experience tank
+execute if score @s[tag=gm4_lt_experience] gm4_lt_value matches 1.. if block ~ ~ ~ hopper{CustomName:"{\"text\":\"Experience Tank\"}",Items:[{Slot:0b,tag:{Enchantments:[{id:"minecraft:mending"}]}}]} run function mending_tanks:check_item
```

Comment:
This is called from liquid tanks so it shouldn't need to check for the name of the block. That will cause problems when an update changes it (eg. using translation keys).

Reply:
Tell that to special. If he removes those checks, I will do so too.